guides/doc-Upgrading_and_Updating/topics/upgrading_external_database.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Upgrading_External_Database_{context}"]
+= Upgrading External Database
+
+You can upgrade an external database from {RHEL} 7 to {RHEL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+Create a new {RHEL} 8 based host for PostgreSQL server that follows the external database on {RHEL} 8 documentation.
+
+.Procedure
+. Copy the backup database to the new host.
```

Comment:
I am thinking we should mention as part of prerequisites that they will need to take a backup as part of this process and should consider a maintenance window to do so.